lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -1010,6 +1010,170 @@ module Resources
     }.freeze
 
     R4_RELATEDPERSON_CREATE ||= {
+      'resourceType': 'RelatedPerson',
+      'extension': [
+        {
+          'valueCodeableConcept': {
+            'coding': [
+              {
+                'system': 'http://hl7.org/fhir/resource-types',
+                'code': 'Patient',
+                'display': 'Patient'
+              }
+            ],
+            'text': 'Patient'
+          },
+          'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/relationship-level'
+        }
+      ],
+      'identifier': [
+        {
+          'id': 'CI-490060520-1',
+          'use': 'usual',
+          'type': {
+            'coding': [
+              {
+                'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/4',
+                'code': '22',
+                'display': 'Military Id',
+                'userSelected': true
+              }
+            ],
+            'text': 'Military Id'
+          },
+          'system': 'urn:oid:2.16.840.1.113883.3.42.10001.100001.12',
+          'value': '1000000105'
+        }
+      ],
+      'active': true,
+      'patient': {
+        'reference': 'Patient/12724068',
+        'display': 'SMART, HAILEY'
+      },
+      'relationship': [
+        {
+          'id': 'CI-490017041-0',
+          'extension': [
+            {
+              'valuePeriod': {
+                'start': '2019-12-26T16:06:26.000Z'
+              },
+              'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/period'
+            },
+            {
+              'valueCodeableConcept': {
+                'coding': [
+                  {
+                    'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/40',
+                    'code': '156',
+                    'display': 'Mother',
+                    'userSelected': true
+                  },
+                  {
+                    'system': 'http://terminology.hl7.org/CodeSystem/v3-RoleCode',
+                    'code': 'MTH',
+                    'display': 'mother',
+                    'userSelected': false
+                  }
+                ],
+                'text': 'Mother'
+              },
+              'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/relation'
+            }
+          ],
+          'coding': [
+            {
+              'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/351',
```

Comment:
Now we support standard codes can you add standard codes every where in place of proprietary codes 
@SantanuBanerjee need your input here.

Reply:
I think there is a validation for write, that restricts us to send both standard and proprietary together for a write workflow?

Reply:
updated accordingly: 
https://github.com/cerner/fhir.cerner.com/pull/689/commits/6ef20fc0201eeb01007296ad539296c4bcf3250e#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eL1066-L1071

https://github.com/cerner/fhir.cerner.com/pull/689/commits/6ef20fc0201eeb01007296ad539296c4bcf3250e#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eL1085-L1090

https://github.com/cerner/fhir.cerner.com/pull/689/commits/6ef20fc0201eeb01007296ad539296c4bcf3250e#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eL1156-L1161

Reply:
Added a comment on the commit https://github.com/cerner/fhir.cerner.com/commit/6ef20fc0201eeb01007296ad539296c4bcf3250e#r53173965